examples/svm/plot_custom_kernel.py

```diff
@@ -12,7 +12,7 @@
 import matplotlib.pyplot as plt
 from sklearn import svm, datasets
 
-# import some data to play with
+# Generate iris dataset
```

Comment:
I prefered the previous comment as we do not "generate" the data, we just load an arbitrary dataset from disk.